include/ukv/cpp/db.hpp

```diff
@@ -157,7 +157,7 @@ class context_t : public std::enable_shared_from_this<context_t> {
                 continue;
             return ukv_collection_t(*id_it);
         }
-        return ukv_collection_t(ukv_collection_main_k);
+        return status_t {"Collection not found."};
```

Comment:
This can't be done like that, as this error message would be deallocated to the underlying engine allocator, where it will fail.